### DISCUSSION TITLE
My repository can not recognize the language

### DISCUSSION THREAD

**Discussion by DrAugus** [community/community#23116] [New to GitHub]
<p>All my files were not recognized, like ts, js, vue  <a href="https://github.com/DrAugus/Augus" rel="noopener nofollow ugc">click</a></p>

**Comment by jfullstackdev** [comment]
<p>maybe this one</p>
<p><a href="https://stackoverflow.com/questions/34713765/github-changes-repository-to-the-wrong-language" class="onebox" target="_blank" rel="noopener">https://stackoverflow.com/questions/34713765/github-changes-repository-to-the-wrong-language</a></p>

**Comment by DrAugus** [comment]
<p>I tried before. It’s still no effect now. Maybe folder <code>.vuepress</code> excluded…</p>

**Comment by jfullstackdev** [comment, marked as answer]
<p>try asking the question here</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://github.com/github/linguist">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">

      <a href="https://github.com/github/linguist" target="_blank" rel="noopener">GitHub</a>
  </header>

  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/345;"><img src="https://user-images.githubusercontent.com/47092464/181074934-3e044b79-7e96-4d70-b0f7-6fb236650e72.png" class="thumbnail" width="690" height="345" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/5/1/513b9848e8914a255b2fd05909685eb8d6ca5b1b_2_690x345.png, https://aws1.discourse-cdn.com/github/optimized/3X/5/1/513b9848e8914a255b2fd05909685eb8d6ca5b1b_2_1035x517.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/5/1/513b9848e8914a255b2fd05909685eb8d6ca5b1b.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/5/1/513b9848e8914a255b2fd05909685eb8d6ca5b1b_2_10x10.png"></div>

<h3><a href="https://github.com/github/linguist" target="_blank" rel="noopener">GitHub - github/linguist: Language Savant. If your repository's language is...</a></h3>

  <p>Language Savant. If your repository's language is being reported incorrectly, send us a pull request! - GitHub - github/linguist: Language Savant. If your repository's language is being rep...</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>
